proto/vendor/tendermint-celestia-mods/abci/types.proto

```diff
@@ -0,0 +1,165 @@
+syntax = "proto3";
```

Comment:
as I understand files in celestia-mods are modified by hand. Could you please write some note in readme / echo in update-proto-vendor what steps should be taken to maintain them?